python/spglib/spglib.py

```diff
@@ -139,10 +139,267 @@ def __iter__(self) -> Iterator[str]:
         return iter(dataclasses.asdict(self))
 
 
+@dataclasses.dataclass(eq=False, frozen=True)
+class SpglibDataset(DictInterface):
+    """Spglib dataset information.
+
+    .. versionadded:: 1.9.4
+        The member 'choice' is added.
+    .. versionadded:: 2.5.0
+    """
+
+    number: int
+    """International space group number"""
+    hall_number: int
+    """Hall number.
+
+    This number is used in
+        :func:`get_symmetry_from_database` and
+        :func:`get_spacegroup_type`.
+    """
+    international: str
+    """International symbol"""
+    hall: str
+    """Hall symbol"""
+    choice: str
+    """Centring, origin, basis vector setting"""
+    transformation_matrix: NDArray[np.double]
+    """Transformation matrix from input lattice to standardized lattice.
+
+    shape=(3, 3), order='C', dtype='double'
+
+    .. code-block::
+
+        L^original = L^standardized * Tmat.
+
+    See the detail at :ref:`dataset_origin_shift_and_transformation`.
+    """
+    origin_shift: NDArray[np.double]
+    """Origin shift from standardized to input origin.
+
+    shape=(3,), dtype='double'
+
+    See the detail at :ref:`dataset_origin_shift_and_transformation`.
+    """
+    rotations: NDArray[np.intc]
+    """Matrix (rotation) parts of space group operations.
+
+    shape=(n_operations, 3, 3), order='C', dtype='intc'
+
+    Space group operations are obtained by
+
+    .. code-block:: python
+
+        [(r,t) for r, t in zip(rotations, translations)]
```

Comment:
Put this in a `..note::` or higher.

Similar for `translations`

Reply:
Done
![image](https://github.com/spglib/spglib/assets/7272505/09635882-2783-49e1-aa3d-15a99b1bda78)